db/db_impl/db_impl_merge.cc

```diff
@@ -307,6 +307,13 @@ Status DBImpl::MergeDisjointInstances(const MergeInstanceOptions& merge_options,
   {
     autovector<autovector<VersionEdit*>> edit_ptrs;
     autovector<const MutableCFOptions*> cf_mopts;
+    // Mark the version edits as an atomic group if the number of version edits
+    // exceeds 1.
+    if (num_cfs > 1) {
+      for (size_t i = 0; i < num_cfs; i++) {
+        cf_edits[i].MarkAtomicGroup(num_cfs - i - 1 /*remaining_entries*/);
```

Comment:
What if the edit is empty?

Reply:
It's fine. It will still be encoded with a special trailing field.

Reply:
Or perhaps we can remove all those empty records before proceed. For example, if all data are in writecf, then only one edit is needed. For rawkv, all data are in default cf.

And if the edit contains too many files, will it be marked as atomic group by itself?